src/spawn.cpp

```diff
@@ -201,11 +201,10 @@ void Spawns::startup()
 		return;
 	}
 
-	for (Npc* npc : npcList) {
+	for (auto npc : npcList) {
```

Comment:
Could this be `const auto&`
```suggestion
	for (const auto& npc : npcList) {
```